.pre-commit-config.yaml

```diff
@@ -22,10 +22,19 @@ repos:
     rev: v1.5
     hooks:
     -   id: autopep8
--   repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21
+-   repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21-2
     hooks:
     -   id: isort
+        args: [--diff]
+        types: [file, python, text]
+        verbose: true
+-   repo: https://github.com/timothycrosley/isort
```

Comment:
The mirror we were using for the `isort` hook said it was deprecated. It also really bothered me that when `isort` would fail, you wouldn't actually see the modification that was made (and ergo the reason why your build failed on `isort`).